src/libraries/System.Runtime.Serialization.Formatters/src/System/Runtime/Serialization/SerializationGuard.cs

```diff
@@ -0,0 +1,23 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+using System.Reflection;
+
+namespace System.Runtime.Serialization
+{
+    internal sealed class SerializationGuard
+    {
+        private static readonly MethodInfo? s_startDeserialization =
```

Comment:
This is ok for now but should probably be removed before RTM. Serialization Guard has been a contentious feature and there's no guarantee it will retain its current API shape (or even exist at all!) going forward. Private reflection from an OOB package back into the shared framework adds unnecessary risk.

Reply:
I agree. Can we just remove the whole thing, both from here and anywhere that's enlisting in seriakization guard?

Reply:
I'll remove the invocation from here, since Levi's already cool with that.  I'll let the "can we delete SG altogether" fight continue elsewhere, since BinaryFormatter isn't the only thing activating it.